src/platform/qt/ShaderSelector.cpp

```diff
@@ -66,33 +89,47 @@ void ShaderSelector::selectShader() {
 #if !defined(USE_LIBZIP) && !defined(USE_MINIZIP)
 	QString name = GBAApp::app()->getOpenDirectoryName(this, tr("Load shader"), path.absolutePath());
 #else
-	QString name = GBAApp::app()->getOpenFileName(this, tr("Load shader"), "mGBA Shaders (*.shader)", path.absolutePath());
+	QString name = GBAApp::app()->getOpenFileName(this, tr("Load shader"), "mGBA Shaders (*.shader manifest.ini)", path.absolutePath());
```

Comment:
Hmm... This really ought to be `QStringLiteral("%1 (*.shader manifest.ini)").arg(tr("mGBA Shaders"))`. Might as well work that in.